posts/2023-06-09-setting-up-a-home-energy-monitoring-system.adoc

```diff
@@ -0,0 +1,711 @@
+
+We see that the power at which the battery is charged rises from an initial value of ±160W to 180W over a period of almost 2 hours. I think this is because it wasn't very warm outside, the battery's temperature rises during charging and the charging current is probably a function of that temperature, but that's an unconfirmed and personal theory. After reaching the peak, the charging slows down and stops after another one and a half hour.
+
+In conclusion, we can learn a lot about the different electrical consumers in the house and even learn a few special things about certain devices like the air conditioning.
+
+=== The Power Quality dashboard
+
+This dashboard contains two graphs that display the following data for all channels on the _Smappee Infinity_:
+
+* Apparent and Real power 
+* cos(&#966;)
+
+We can have a look at the data for the garage when the hybrid car is charged:
+
+image::charging-hybrid-car-power.png[Power consumption during charging of hybrid car]
+
+The apparent power and real power are 2365 VA and 2357 W respectively. This corresponds to a cos(&#966;) = 0.997 which is pretty much close to ideal. This is especially important as the power consumed during the charging is about 2.4kW. 
+
+On the other hand, the charger of our electrical bicycle charger's cos(&#966;) is only 0.69. So, when charging, the different powers are P~real~ = 180W, P~a~ = 260VA, and P~reactive~ = 187W. Even though it's "only 187W", when thousand or millions of devices with a low Power factor are online, the impact _is_ significant.
+
+<<<
+[id=cos-phi]
+=== Electrical Power Factor (also known as cos(&#966;))
+
+The Power Factor is a measure of an electrical system's energy utilisation efficiency. The power consumption can be decomposed in three parts:
+
+* P~a~: Apparent power, which is the product of the measured voltage and current.
+
+* P~real~: Effective- or Real Power is the part that actually produces work in the broad sense of the word, it's not the internal efficiency of the device itself. For example, an incandescent lightbulb consuming 100W converts 98W to heat and only 2W to light. So, in this case, the Productive Power is 100W (this is not to be confused with the efficiency of that bulb which is 2%).
+
+* P~reactive~: Reactive power (VAR - Volt-Ampère-Reactive) which is the part that doesn't perform any work, but that still results in energy flowing between the electricity producer and consumer.
+
+The relation between these components is the following:
+
+* P^2^~a~ = P^2^~real~ + P^2^~reactive~
+
+* P~real~ = P~a~ . cos(&#966;)
+
+* P~reactive~ = P~a~ . sin(&#966;)
+
+(If you're interested about the theory around this topic, read link:../media/2023-05-05-setting-up-a-home-energy-monitoring-system-power-factor.pdf[this article] about it.)
+
+In electrical systems, cos(&#966;) is a value between 0 and 1. When the reactive power is 0, cos(&#966;) is equal to 1 and P~a~ and P~real~ have equal values.
```

Comment:
```suggestion
In electrical systems, cos(&#966;) is a value between 0 and 1. When the Reactive power is 0, cos(&#966;) is equal to 1 and P~a~ and P~real~ have equal values.
```
Should this be `reactive` or `Reactive`?

Reply:
For consistency, going of upper caps version